lib/v2/screens/explore_screens/vote_screens/proposal_details/interactor/viewmodels/proposal_details_bloc.dart

```diff
@@ -23,17 +24,24 @@ class ProposalDetailsBloc extends Bloc<ProposalDetailsEvent, ProposalDetailsStat
       List<Result> results = await GetProposalDataUseCase().run(state.proposals[state.currentIndex + 1]);
       yield NextProposalDataStateMapper().mapResultsToState(state, results);
     }
-    if (event is OnFavourButtonTapped) {
-      yield state.copyWith(voteChoice: VoteChoice.favour);
+    // if (event is OnFavourButtonTapped) {
```

Comment:
Are we going to use this in the feature?

Reply:
addressed